program/tests/tests/limits.rs

```diff
@@ -67,7 +67,7 @@ async fn test_max_validators_maintainers() {
 
     // The maximum number of validators that we can support, before Deposit or
     // StakeDeposit fails.
```

Comment:
Let’s also test `Unstake` here now then, to make sure that that’s not the limiting factor.